### ISSUE TITLE
FR: Add schema markup?

### ISSUE THREAD

**Issue by RCheesley** [suresoftware/october-power-seo#8]
I came across this Gist when looking at how to implement schema markup, and it got me thinking, I wondered if it might be feasible to add schema markup in the same way as the content fields are added?

https://gist.github.com/mbaxter91288/216d478068af9be5bdc234d8aefb7e40

It would be a neat feature (perhaps having an option in global settings to set the 'default' would be helpful, with an option per-resource to specify or change as appropriate)?

**Comment by Humni** [comment]
Yes, this is something I've been looking into for quite some time. I've done a lot of work with structured data and there are quite a few complexities in making sure that it works well across different OctoberCMS sites. 

It's possible that it fits better into a separate plugin, and is designed to be highly extensible so that other plugins can integrate with it. I'll do some planning and see how much work it'll be to get something with basic functionality working. 

Specifically what types of structured data are you wanting? https://schema.org is the formal definition but Google has some really nice documentation around it also.

**Comment by RCheesley** [comment]
Yeah, I've done a fair amount, mostly using JSON-LD markup.

I think going with Google's documentation is a good route, I'd expect you'd want the basics that are in the gist I shared to describe the organisation and relations (e.g. social profiles, etc) and website, article, blog, forum, contact/address/location etc.
